### ISSUE TITLE
Visual whitepoint editor does not work

### ISSUE THREAD

**Issue by ledoge** [eoyilmaz/displaycal-py3#169]
When I set "Whitepoint" to "Chromaticity Coordinates" in the Calibration tab and click the icon to open the visual whitepoint editor, I get the following error:

```
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/DisplayCAL/display_cal.py", line
6623, in visual_whitepoint_editor_handler
    ) or display_name.startswith(b"Chromecast "):
TypeError: startswith first arg must be str or a tuple of str, not bytes
```
Unfortunately, making it work is not as easy as just changing the byte strings to strings there, as I got a wx-related error when I tried that.

**Comment by p5k369** [comment]
Hey @ledoge thanks for finding those bugs. The first one you solved by yourself, the second one is completely unrelated to the first one, take a look at #175 if you want to try it locally or wait until it gets merged into dev.

**Comment by p5k369** [comment]
Fixed in #175
